src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+/*#region defaultSettings*/
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+}
+/*#endregion*/
+
+/*#region Header*/
+.site-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
```

Comment:
Check this out
![Screenshot 2022-05-10 at 22 37 35](https://user-images.githubusercontent.com/75496546/167708467-40acedaf-3aff-4e4e-9029-a9c424d0ddb0.png)